app/views/layouts/forever_style_guide/application.html.erb

```diff
@@ -14,7 +14,11 @@
   <%= csrf_meta_tags %>
 </head>
 <body class="style-guide" data-spy="scroll" data-target=".style-guide-scrollspy">
-  <%= render partial: "layouts/forever_style_guide/navigation" %>
+
+  <% if !content_for?(:hide_nav) %>
```

Comment:
I see what you are doing with this, I think it would be better to use a local variable similar to what we actually have in www: https://github.com/forever-inc/forever-www/blob/dev/app/views/layouts/_navigation.html.erb

Then you would pass this variable in via the controller route.